### PR TITLE
feat: add directions to order generation flow (FPLA-1105)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-1105

### Change description ###

New directions free text added to order flow. Directions appear in generated document but do not appear in tab view.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/FurtherDirections.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class FurtherDirections {
+    private final String directionsNeeded;
```
Comment (akomisarek) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/FurtherDirections.java]: I know you started doing the JsonGetters / Setters could we model this as boolean and use the similar fashion to model it as boolean? :)

Comment (swalker125) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/FurtherDirections.java]: do something like if directionsNeeded = yes, set, if no, set as empty string? definitely possible

Comment (swalker125) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/FurtherDirections.java]: is it better practice to do optional logic on serialisation?

Comment (swalker125) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/FurtherDirections.java]: @akomisarek

**Comment by joez-f** [comment]
do we still want to merge 'use-IDAM-preview' as part of this PR? I believe IDAM AAT issues are resolved now - also we can open PR to https://github.com/hmcts/rdo-docmosis-templates development and approve it ourselves (so we can get this into QA stage), just not into prod

**Pull request closed without merging**